elemica-suggest.coffee

```diff
@@ -154,10 +160,11 @@ elemicaSuggest 0.7.1-SNAPSHOT.
         $(this).attr 'autocomplete', 'off'
 
         $(this).on 'blur', (event) ->
-          removeSuggestions event.target
+          $target = $(event.target)
+          removeSuggestions $target
 
           if $valueInput.val() == ""
-            $(event.target).val("")
+            $target.val("") if noSuggestionMatched($target.val())
             afterSelect(null)
```

Comment:
Nope :) I'm saying that maybe it's better to leave it as it is because user may want to:
1. clear typeahead input - needs to set `noSuggestionMatched: -> false`
2. run code in `afterSelect`

If we were running `afterSelect` depending on result of `noSuggestionMatched`, 2. wouldn't happen.

What do you think?